src/Microsoft.PowerShell.Commands.Utility/resources/WebCmdletStrings.resx

```diff
@@ -199,7 +199,7 @@
     <value>The cmdlet cannot run because the following parameter is missing: Proxy. Provide a valid proxy URI for the Proxy parameter when using the ProxyCredential or ProxyUseDefaultCredentials parameters, then retry.</value>
   </data>
   <data name="ReadResponseComplete" xml:space="preserve">
-    <value>Reading web response stream completed. Downloaded: {0}</value>
+    <value>Reading web response stream completed. Bytes downloaded: {0}</value>
```

Comment:
Again bytes? :-)

Reply:
It's the last message, from the completion progress record, so I think it's better to report the exact size of the package.
The string returned by `DisplayHumanReadableFileSize` is a rough number, not accurate.